byterun/floats.c

```diff
@@ -530,6 +650,49 @@ CAMLprim value caml_log1p_float(value f)
   return caml_copy_double(caml_log1p(Double_val(f)));
 }
 
+#ifndef HAS_C99_FLOAT_OPS
+static inline double simple_erf(double x)
+{
+  double a1 =  0.254829592, a2 = -0.284496736, a3 = 1.421413741,
+         a4 = -1.453152027, a5 =  1.061405429, p  = 0.3275911,
+         t, y;
+
+  int sign = (x >= 0) ? 1 : -1;
+  x = fabs(x);
+  t = 1.0 / (1.0 + p * x);
+  y = 1.0 - (((((a5 *t  + a4) * t) + a3) * t + a2) * t + a1) * t * exp(-x * x);
```

Comment:
Does that line not mean that `simple_erf` is using an asymptotic extension for erfc then using 
erf = 1 - erfc for recovering erf, and then implements `simple_erfc` as `1 - simple_erf`?
Would it not be better to define `simple_erfc` directly here and then implement `simple_erf` as `1 - simple_erfc`?
Or is this round-trip and the associated loss of numerical precision somehow beneficial in this case?